### PR TITLE
Method into an array of methods

### PR DESCRIPTION
**Pull request by liamkeaton**
## Feature Description
Methods should be an array rather than just one method

### CODE REVIEW

File: utils/payment.js [hunk 1]
```diff
@@ -140,12 +142,13 @@ class Payment {
 	/**
 	 * Return the requested payment method
 	 * @param {string} name `apple` is only supported currently
+	 * @return {array<object>}
 	 */
 	static getPaymentMethod (name) {
```
Comment (remybach) [inline utils/payment.js]: Should the function be named `getPaymentMethods` to reflect the change?

Comment (liamkeaton) [reply utils/payment.js]: It only returns one, although it is array

Comment (remybach) [reply utils/payment.js]: 🤔 That feels kinda weird to be honest... why does it have to be an array then? Definitely worth explaining all of this in the JSDoc for the function at the very least.

Comment (liamkeaton) [reply utils/payment.js]: Yup

**Pull request merged**